From f7b9e15bc5e03d6a190e7752d4c57c55849af719 Mon Sep 17 00:00:00 2001
From: Ron Eldor <Ron.Eldor@arm.com>
Date: Sun, 1 Jul 2018 11:27:10 +0300
Subject: [PATCH 15/43] Fix compilation error

Fix compilatin error that happened after rebasing.
Change `mbedtls_zeroize()` to `mbedtls_platfrom_zeroize()`.
---
 library/ssl_tls.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library/ssl_tls.c b/library/ssl_tls.c
index a21ac92b4..e037266da 100644
--- a/library/ssl_tls.c
+++ b/library/ssl_tls.c
@@ -9133,7 +9133,7 @@ void mbedtls_ssl_free( mbedtls_ssl_context *ssl )
 #endif
 
 #if defined (MBEDTLS_SSL_DTLS_SRTP)
-    mbedtls_zeroize( ssl->dtls_srtp_info.dtls_srtp_keys, ssl->dtls_srtp_info.dtls_srtp_keys_len );
+    mbedtls_platform_zeroize( ssl->dtls_srtp_info.dtls_srtp_keys, ssl->dtls_srtp_info.dtls_srtp_keys_len );
  //   mbedtls_free( ssl->dtls_srtp_keys );
 #endif /* MBEDTLS_SSL_DTLS_SRTP */
 
-- 
2.17.1

